### ISSUE TITLE
More data in DALIClassificationIterator other than ["data", "label"]

### ISSUE THREAD

**Issue by pawopawo** [NVIDIA/DALI#3167]
How to ["data1", "data2", "label"]  dataloader ?

**Comment by mzient** [comment]
@pawopawo You have to use `DALIGenericIterator` - DALIClassificationIterator is just that - image and label.
See https://docs.nvidia.com/deeplearning/dali/user-guide/docs/plugins/pytorch_plugin_api.html for more details.
